openhands/resolver/resolve_issue.py

```diff
@@ -581,14 +633,24 @@ def int_or_none(value):
             f'ghcr.io/all-hands-ai/runtime:{openhands.__version__}-nikolaik'
         )
 
-    owner, repo = my_args.repo.split('/')
-    token = my_args.token if my_args.token else os.getenv('GITHUB_TOKEN')
-    username = my_args.username if my_args.username else os.getenv('GITHUB_USERNAME')
+    parts = my_args.repo.split('/')
+    if len(parts) > 2:
+        owner = '/'.join(parts[:-1])
+        repo = parts[-1]
+    else:
+        owner, repo = parts
+
+    token = my_args.token or os.getenv('GITHUB_TOKEN') or os.getenv('GITLAB_TOKEN')
+    username = my_args.username if my_args.username else os.getenv('GIT_USERNAME')
     if not username:
-        raise ValueError('Github username is required.')
+        raise ValueError('username is required.')
 
     if not token:
-        raise ValueError('Github token is required.')
+        raise ValueError('token is required.')
+
+    platform = identify_token(token)
+    if platform == Platform.INVALID:
+        raise ValueError('token is invalid.')
```

Comment:
```suggestion
        raise ValueError('Token is invalid.')
```

This is really nice!